### PR TITLE
Update errors and Swagger docs

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -125,13 +125,15 @@ app.use(function(err, req, res, next) {
         .json({ message: err.data[0].message, path: err.data[0].path });
     } else {
       // unexpected error, so log it
-      logger.error({ url: req.originalUrl, message: err.message });
+      logger.debug(err);
```
Comment (avarsava) [inline app.js]: Why the change in severity level for logging?

Comment (callunity) [reply app.js]: Thanks for catching this! I meant to add additional logging at the `debug` level (which will only be done in non-prod environments, as configured in `utils/logger.js`) and have reverted it so that it still writes to `logger.error` in all environments.
